src/test/java/com/rackspacecloud/metrics/influxdbscaler/InfluxDBHelperTests.java

```diff
@@ -0,0 +1,108 @@
+package com.rackspacecloud.metrics.influxdbscaler;
+
+import com.rackspacecloud.metrics.influxdbscaler.collectors.InfluxDBHelper;
+import com.rackspacecloud.metrics.influxdbscaler.models.StatsResults;
+import com.rackspacecloud.metrics.influxdbscaler.providers.StatefulSetProvider;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class InfluxDBHelperTests {
+	private final static String SHOW_STATS = "q=SHOW STATS";
+
+	RestTemplate restTemplateMock;
+
+	@MockBean
+	StatefulSetProvider statefulSetProviderMock;
+
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
```

Comment:
The `SpringRunner` and `@SpringBootTest` already trigger the mock annotation processing.

Reply:
Done